_ingest-pipelines/processors/drop.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Drop
+parent: Ingest processors
+nav_order: 70
+---
+
+# Drop processor
+
+The `drop` processor is used to discard documents without indexing them. This is useful for preventing documents from being indexed based on certain conditions. For example, you might use a `drop` processor to prevent documents that are missing important fields or contain sensitive information from being indexed. 
```

Comment:
```suggestion
The `drop` processor is used to discard documents without indexing them. This can be useful for preventing documents from being indexed based on certain conditions. For example, you might use a `drop` processor to prevent documents that are missing important fields or contain sensitive information from being indexed. 
```